### PR TITLE
Camera detection to add in SplineCurve

### PR DESCRIPTION
**Pull request by Danilodum**

### CODE REVIEW

**Comment by Danilodum** [comment]
please review @SDumasy @jonathanraes @olivier-dikken

File: src/main/java/com/funkydonkies/camdetect/VideoCap.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    private VideoCapture cap;
+    private Mat2Image mat2Img = new Mat2Image();
+    private static final int INPUTSOURCENUMBER = 1;
+    private boolean camOn = false;
+
+    /**
+     * 
+     */
+    VideoCap() {
+        cap = new VideoCapture(INPUTSOURCENUMBER);
```
Comment (Danilodum) [inline src/main/java/com/funkydonkies/camdetect/VideoCap.java:25]: Words should be divided by an underscore

**Comment by Danilodum** [comment]
Removal of `AllTests.java` makes `testSpawnUsingDefaults()` in `BallTest.java` fail. Makes 0 sense. Commenting test to see if the build passes.

**Comment by ghost** [comment]
fixed some check style errors and bugs, still 1 bug remaining

**Comment by Danilodum14** [comment]
fixed bug

File: src/main/java/com/funkydonkies/camdetect/Mat2Image.java [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        return img;
+    }
+        
+    /**
+     * Method comes from implementing the bridge interface.
+     * used by classes outside of camdetect package to access the control points.
+     * @return float array containing the control points
+     */
+	public float[] getControlPoints() {
+		return Arrays.copyOf(interestPoints, numPoints);
```
Comment (Danilodum) [inline src/main/java/com/funkydonkies/camdetect/Mat2Image.java:182]: fixed bug (l. 182)

**Pull request merged**